src/test/regress/input/multi_copy.source

```diff
@@ -107,6 +107,15 @@ SELECT count(*) FROM customer_copy_hash;
 -- Confirm that data was copied
 SELECT count(*) FROM customer_copy_hash;
 
+-- Make sure that master_update_shard_statistics() only updates shard length for
```

Comment:
> For range tables, it is a bit tricky because we expect the user to set min/max values. I think we also should not update min/max values on range-partitioned tables? 

Should we add that to the check then?